### PR TITLE
OPENHELPD-72 Initial dockerise

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@miltontony @JayH5 review please

**Comment by miltontony** [comment]
looks good 👍

File: casepro/settings_production.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+HOSTNAME = os.environ.get('HOSTNAME', 'localhost:8000')
+
+SITE_API_HOST = os.environ.get('SITE_API_HOST', 'http://localhost:8001/')
+
+DATABASES = {
+    'default': dj_database_url.config(
+        default=os.environ.get(
+            'CASEPRO_DATABASE',
+            'postgres://casepro:nyaruka@localhost/casepro')),
```
Comment (JayH5) [inline casepro/settings_production.py]: Is this a plaintext password? I guess localhost but still :/

File: casepro/settings_production.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+from __future__ import unicode_literals
+import os
+
+# import our default settings
+from settings_common import *  # noqa
+
+SECRET_KEY = os.environ.get('SECRET_KEY', 'REPLACEME')
+if os.environ.get('DEBUG', 'False') == 'True':  # envvars are strings
```
Comment (JayH5) [inline casepro/settings_production.py]:
```python
DEBUG = os.environ.get('DEBUG', 'False') == 'True'
```
?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,4 @@
+FROM praekeltfoundation/django-bootstrap
+ENV DJANGO_SETTINGS_MODULE "casepro.settings_production"
+RUN ./manage.py collectstatic --noinput
+ENV APP_MODULE "hellomama_registration.wsgi:application"
```
Comment (JayH5) [inline Dockerfile]: `hellomama_registration`?

File: casepro/settings_production.py (hunk 2 above)
Comment (imsickofmaps) [inline casepro/settings_production.py]: the second param is the default if the envvar isn't present. Make more sense now?

File: casepro/settings_production.py (hunk 1 above)
Comment (imsickofmaps) [inline casepro/settings_production.py]: it's their standard base config, we'll always override it in production

**Pull request merged**